.vscode/launch.json

```
@@ -8,7 +8,6 @@
             "name": "Attach",
             "type": "cppvsdbg",
             "request": "attach",
-            "processName": "FlightSimulator.exe",
```

```suggestion
            "processId": "${command:pickProcess}",
```
You can use this instead.

---

That is actually the default, but we can make it explicit I guess.

---

https://code.visualstudio.com/docs/cpp/launch-json-reference#_additional-properties